scripts/create_scan_config.sh

```diff
@@ -17,8 +17,9 @@ IMAGE=${1}
 echo "Creating security scan configuration file:"
 cat <<EOF | tee ${FILENAME}
 module-name: btp-operator
+rc-tag: ${IMAGE}
 protecode:
-  - ${IMAGE}
+  - europe-docker.pkg.dev/kyma-project/prod/btp-manager:${IMAGE}
```

Comment:
Is there a simple way to pass it to the script (to reduce the number of places where repository location is hard-wired). If this requires too much hassle, leave it as it is.

Reply:
I could pass the path to the image as an argument. Also the `{IMAGE}` will be replaced with `{TAG}`

Reply:
We have already an env `IMAGE_REPO: europe-docker.pkg.dev/kyma-project/prod/btp-manager` so it doesn't requieres too much hassle.